fintech-examples/fintech-api/src/main/java/de/adorsys/opba/fintechapi/resource/AccountInformationResource.java

```diff
@@ -0,0 +1,7 @@
+package de.adorsys.opba.fintechapi.resource;
+
+import org.springframework.stereotype.Controller;
+
+@Controller
```

Comment:
As we are aiming to REST it is better to use RestController annotation instead of `@Controller`